templates/store/_category-partial.html

```diff
@@ -13,7 +19,7 @@
           {% if loop.index != 1 or snap["icon_url"] == "" %}
               {% if has_featured %}
                   {% set icon_size = "large" %}
-                  <div class="col-4 col-medium-3">
+                  <div class="col-4 col-medium-3 col-small-2">
```

Comment:
`col-small-2` breaks how snaps are displayed on mobile:

<img width="279" alt="Screenshot 2019-08-16 at 09 05 38" src="https://user-images.githubusercontent.com/83575/63149733-40709400-c005-11e9-8ec4-04d549537ca1.png">

Reply:
And funny how 'featured banner' on mobile seems to be smaller and less informative than snaps in regular listing.

Reply:
Ah I meant to back that change out - I was trying something for a follow-up PR, will remove.
The old way these icons worked is they became smaller at the medium breakpoint so I was testing that they looked ok next to the smaller banner size on mobile, but didn't want to do the PR as part of this one!